--- /home/rachneet/PycharmProjects/rf_classification/cnn_model.py
+++ /home/rachneet/PycharmProjects/rf_classification/cnn_model.py
@@ -87,9 +87,7 @@
         self.softmax = nn.Softmax(dim=1)
         self.sigmoid = nn.Sigmoid()
 
-        if filters == 64 and n_fc_neurons == 128:
-            self._create_weights(mean=0.0, std=1)
-        elif filters == 256 and n_fc_neurons == 1024:
+        if filters == 256 and n_fc_neurons == 1024:
             self._create_weights(mean=0.0, std=0.05)
         elif filters == 1024 and n_fc_neurons == 2048:
             self._create_weights(mean=0.0, std=0.02)